# The environment information
#
Bounds bounds
obstacle[] obstacles